patches/chrome-browser-ui-views-profiles-avatar_toolbar_button.cc.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
+index a053f001493dc3815ee9bb0a6d4526bfe65718dd..348a783485741370796073076cbbf78666911665 100644
+--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
++++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.cc
```

Comment:
I prefer a subclass for the tooltip patch since I envision multiple different types of things here in the future.